src/themes/shared/typography.ts

```diff
@@ -8,97 +8,97 @@ export const typography = {
   display1: {
     fontFamily: fonts.makeItSans,
     fontWeight: fontWeights.bold,
-    lineHeight: lineHeights.display,
+    lineHeight: lineHeights.xs,
     fontSize: fontSizes['5xl'],
   },
   display2: {
     fontFamily: fonts.makeItSans,
     fontWeight: fontWeights.bold,
-    lineHeight: lineHeights.display,
+    lineHeight: lineHeights.xs,
     fontSize: fontSizes['4xl'],
   },
   display3: {
     fontFamily: fonts.makeItSans,
     fontWeight: fontWeights.bold,
-    lineHeight: lineHeights.display,
+    lineHeight: lineHeights.xs,
     fontSize: fontSizes['3xl'],
   },
   display4: {
     fontFamily: fonts.makeItSans,
     fontWeight: fontWeights.bold,
-    lineHeight: lineHeights.display,
+    lineHeight: lineHeights.xs,
     fontSize: fontSizes['2xl'],
   },
   heading1: {
     fontFamily: fonts.makeItSans,
     fontWeight: fontWeights.bold,
-    lineHeight: lineHeights.display,
+    lineHeight: lineHeights.sm,
```

Comment:
shouldn't be here `xs`, since it was `display`?

Reply:
Yes, the thing is that we adjusted the lineHeight tokens -> https://www.figma.com/file/WmNmJ7jdB0Z7tOjj516nFB/AS24-Tokens?node-id=3%3A3